indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -52,7 +52,7 @@ def get_query_handler(self):
 
         """
         import redis
-        r = redis.Redis(host=self.hostname, port=self.port, db=self.db, socket_timeout=None)
+        r = redis.Redis(host=self.hostname, port=self.port, db=self.db, socket_timeout=10000)
```

Comment:
have u found it to be a common default param?

Reply:
I just changed it actually, I set what I saw in some tutorials